client/src/js/components/bhTagSelect/bhTagSelect.html

```diff
@@ -4,14 +4,13 @@
     ng-class="{ 'has-error' : TagForm.$submitted && TagForm.tagUuids.$invalid }">
 
     <label class="control-label">
-      <span translate>{{ $ctrl.label }}</span>&nbsp;
-      <a style="cursor: pointer;" ng-click="$ctrl.createUpdateTagsModal()" translate>TAG.ADD</a>
```

Comment:
This is used in the "edit lots" modal to quickly add lots on the fly.  Removing it from the component isn't really what we want since it removes it in each instance of the component.

Instead, let's add a binding to the `bhTagSelect.js` controller to add this `<a>` tag or hide it using an `ngIf` statement.  The default case will be to hide it.  It will be something like this:
```js
bindings : {
  // bindings...
  'enableAddTag' : '<?',
  // etc..
}
```

Then you can have a `ng-if="$ctrl.enableAddTag"` on is HTML tag and that is probably the sufficient.